database/things/mips.pldb

```diff
@@ -62,8 +62,10 @@ rijuRepl https://riju.codes/mips
   	.data
   message:
   	.string "Hello, world!\n"
-  
 
+indeedJobs mips engineer
+ 2022 1150
```

Comment:
MIPS is an old architecture. Again, might this be for another type of work?

Reply:
I had looked at the MIPS architecture,  [Link](https://www.indeed.com/jobs?q=MIPS+architecture&l=Texas&vjk=0098240a434d3527)